inst/csv/replacementPatterns.csv

```diff
@@ -401,6 +402,7 @@ impala,#,%temp_prefix%%session_id%
 impala,<hash><hash>,##
 impala,.location,.`location`
 impala,SELECT TOP @([0-9]+)rows @a;,SELECT @a LIMIT @rows;
+impala,NOT NULL,

```

Comment:
Happy to accept this pull request, but I think this rule will cause a lot of trouble on Impala. There are quite a few lines of SQL in OHDSI along the lines of `WHERE x IS NOT NULL` which will break.